libraries/AP_Scripting/examples/copter-fast-descent.lua

```diff
@@ -0,0 +1,153 @@
+-- Copter descends very rapidly in a spiral pattern to a present altitude above home
+--
+-- CAUTION: This script only works for Copter
+-- this script waits for the vehicle to be changed to Guided mode and then:
+--    a) flies a spiral pattern using the velocity and acceleration control
+--    b) slows the spiral and stops at the preset altitude
+--    c) switches to RTL
```

Comment:
I think you should add the URL of the github pull request here. That URL contains nice picture and more info.